source/clear-linux/tutorials/kata.rst

```diff
@@ -43,36 +42,57 @@ Configure Docker to use Kata Containers\* by default
    ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
    EOF
    
-Restart the Docker\* and Kata Containers\* systemd services
+Restart the Docker and Kata Containers systemd services
 
 .. code-block:: bash
 
    sudo systemctl daemon-reload
    sudo systemctl restart docker
 
-Run Kata Containers\*
+Run Kata Containers
 *******************
 
 .. code-block:: bash
 
    sudo docker run -ti busybox sh
 
+.. note::
+
+   In cases where it is necessary to use a proxy server and your proxy
+   environment variables are already set, run the following commands as
+   a shell script to configure Docker:
+
+   .. code-block:: bash
+
+      docker_service_dir="/etc/systemd/system/docker.service.d/"
+      sudo mkdir -p "$docker_service_dir"
+      cat <<EOF | sudo tee "$docker_service_dir/proxy.conf"
+      [Service]
+      Environment="HTTP_PROXY=$http_proxy"
+      Environment="HTTPS_PROXY=$https_proxy"
+      EOF
+      echo "Reloading unit files and starting docker service"
+      sudo systemctl daemon-reload
+      sudo systemctl restart docker
+      sudo docker info
+
 **Congratulations!**
 
-You successfully installed and set up Kata Containers\* on |CLOSIA|.
+You have successfully installed and set up Kata Containers on |CLOSIA|.
 
-More information about Docker\* in |CLOSIA|.
+More information about Docker in |CLOSIA|.
 #############################################
 
-Docker\* on |CLOSIA| provides a docker.service service file to start the Docker\*
-daemon. The daemon will use runc or cc-runtime depending on the environment:
+Docker on |CLOSIA| provides a docker.service service file to start the Docker
+daemon. The daemon will use runc or s/cc-runtime/kata-runtime depending on 
+the environment:
 
-If you are running |CL| on baremetal or on a VM with Nested
-Virtualization activated, Docker\* will use cc-runtime as the default runtime.
-If you are running |CL| on a VM without Nested Virtualization, Docker\*
-will use runc as the default runtime. It is not necessary to configure Docker\*
-to use cc-runtime manually since Docker\* itself will automatically use this
-runtime on systems that support it.
+If you are running |CL| on bare metal or on a VM with Nested
+Virtualization activated, Docker will use s/cc-runtime/kata-runtime as the
+default runtime. If you are running |CL| on a VM without Nested
```

Comment:
I'm sorry, `s/cc-runtime/kata-runtime` means to replace cc-runtime with kata-runtime, and I think you didn't understand it, sorry for the misunderstood 

please remove `s/cc-runtime`

`Virtualization activated, Docker will use kata-runtime as the`

Reply:
PR 192 resolves this. 
https://github.com/clearlinux/clear-linux-documentation/pull/192